### DISCUSSION TITLE
What is the Clippy lint to flag used underscore-prefixed functions?

### DISCUSSION THREAD

**Discussion by laniakea64** [rust-lang/rust-clippy#13256] [Q&A] [answered]
For example:
```rust
fn _foo() {
  println!("Foo");
}

fn main() {
  //_foo();
}
```
Using Rust 1.80.1, this code compiles without warnings and passes Clippy without warnings, and continues to do so after uncommenting the call to `_foo()`.

When the call to `_foo()` is uncommented, I would like Clippy to warn that an underscore-prefixed function is being used.  What is the Clippy lint to flag this?

(Sorry if this is a silly question, but couldn't find the answer myself searching the list of Clippy lints - closest was https://rust-lang.github.io/rust-clippy/stable/index.html#/used_underscore_binding but seems that's only for use of `let _foo = ...`?)

**Comment by flip1995** [comment, marked as answer]
I don't think there currently is a link for that. It could be an extension for the `use_underscore_binding` lint. Even though the lint might have to be renamed or maybe a new lint created (`used_underscore_items`)

**Reply by laniakea64** [reply]
Thanks flip1995 for the answer.  In that case I'll file a new lint suggestion issue.

Edit:  :arrow_right:  https://github.com/rust-lang/rust-clippy/issues/13260
